src/main/java/org/apache/maven/plugins/artifact/buildinfo/ReferenceBuildinfoUtil.java

```diff
@@ -59,17 +58,15 @@
  * Utility to download reference artifacts and download or generate reference buildinfo.
  */
 class ReferenceBuildinfoUtil {
-    private static final Set<String> JAR_TYPES;
+    private static final Set<String> JAR_EXTENSIONS;
 
     static {
         Set<String> types = new HashSet<>();
         types.add("jar");
-        types.add("test-jar");
         types.add("war");
         types.add("ear");
         types.add("rar");
-        types.add("maven-plugin");
```

Comment:
To be consistent, I'd rename `types` to `extensions` as later on, someone will assume there is some mixup here

Reply:
good feedback: done